client/app/bundles/course/assessment/submission/translations.ts

```diff
@@ -417,6 +429,16 @@ const translations = defineMessages({
       'Please do not sign out or close the browser, otherwise \
                     you may have trouble continuing the exam.',
   },
+  timedAssessmentDialogTitle: {
+    id: 'course.assessment.submission.timedAssessmentDialogTitle',
+    defaultMessage: 'You are entering a timed assessment.',
+  },
+  timedAssessmentDialogMessage: {
+    id: 'course.assessment.submission.timedAssessmentDialogMessage',
+    defaultMessage:
+      'You will have {timeLimit, plural, one {# minute} other {# minutes}} in total to finish \
```

Comment:
```suggestion
      'You have {timeLimit, plural, one {# minute} other {# minutes}} to complete \
```

Reply:
I still think that word 'in total' is necessary. If student close the page then reopen, they need to be reminded that the mentioned timeLimit is in total, not from the moment they reopen the page

Reply:
its redundant. Tell them the deadline instead.

Reply:
deadline is also not quite right. As a student, I prefer the system to tell me how many minutes are remaining instead of when will it be due (i.e. prefer the system telling that I have 1 hr left to finish instead of finishing at 19:05:12).

Nevertheless, I got your point and thus I will move the logic of displaying the remaining time in the TimerBanner to be used here as well